scripts/sizeSnapshot/webpack.config.js

```diff
@@ -32,6 +32,11 @@ async function getSizeLimitBundles() {
       webpack: true,
       path: 'packages/material-ui/build/esm/index.js',
     },
+    {
+      name: '@material-ui/core-es',
```

Comment:
What naming convention should we follow? There is precedence with `@material-ui/core/Paper.esm`.

```suggestion
      name: '@material-ui/core.es',
```

By the way, did you change your point of view the "integrity" of the names? Can we skew it to follow a single naming convention?

Reply:
Nope I still want to track these over time. packtracker/packtracker has the same feature and it seems like paularmstrong/build-tracker will have the same feature.

I think `@material-ui/core/es` will fit the best since it's closest to the actual package used.